File: AutoHero.GUI/Events/MessageBox.cs
using AutoHero.WPF.CEF.Events;
using CefSharp;
using CefSharp.ModelBinding;
using System;
using System.Collections.Generic;
using System.Text;
using System.Windows;

namespace AutoHero.CEF.Events
{ 
    public class MessageBox  : IEventRegister
    {
        public BindingOptions binder => BindingOptions.DefaultBinder;

        public bool Show(string message)
        {
            return System.Windows.MessageBox.Show(message) == MessageBoxResult.OK; 
        }
    }
      
}

File: gui-src/src/apis/csharp/events/messageBox.js
/* eslint-disable */

const repository = 'messageBox';
const o = () => window[repository]; 
export const messageBox = {
    show : async (text) => {
      await CefSharp.BindObjectAsync(repository);
      const response = await o.show(text);
      return response;
    }
}